source/installation.txt

```diff
@@ -0,0 +1,186 @@
+.. _cpp-installation:
+
+=============
+Installation
+=============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: running
+
+.. toctree::
+   :titlesonly:
+   :maxdepth: 1
+
+   /installation/windows
+   /installation/macos
+   /installation/linux
+   /installation/advanced
+
+Prerequisites
+-------------
+
+- Any standard Unix platform, or Windows 7 SP1+
+- A compiler that supports C++11 (gcc, clang, or Visual Studio)
+- `CMake <https://cmake.org>`__ 3.2 or later
```

Comment:
Replace "3.2" with "3.15".

This is notably out-dated in the current C++ driver documentation. The minimum required CMake was increased to 3.15. https://github.com/mongodb/mongo-cxx-driver/pull/1116 has been opened to update the C++ driver documentation.